src/net/java/sip/communicator/impl/protocol/jabber/OperationSetContactCapabilitiesJabberImpl.java

```diff
@@ -405,45 +444,80 @@ public void userCapsNodeAdded(String user, String node, boolean online)
      * record for a specific user about the caps node the user has.
      *
      * @param user the user (full JID)
+     * @param fullJids a list of all resources of the user (full JIDs)
+     * @param node the entity caps node#ver
+     * @param online indicates if the user is currently online
+     * @see UserCapsNodeListener#userCapsNodeAdded(String, String, boolean)
+     */
+    public void userCapsNodeRemoved(String user, ArrayList<String> fullJids,
+        String node, boolean online)
+    {
+        /*
+         * It doesn't matter to us whether a caps node has been added or removed
+         * for the specified user because we report all changes.
+         */
+        userCapsNodeChanged(user, fullJids, node, online);
+    }
+
+    /**
+     * Notifies this listener that an <tt>EntityCapsManager</tt> has changed a
+     * record for a specific user about the caps node the user has.
+     *
+     * @param user the user (full JID)
+     * @param fullJids a list of all resources of the user (full JIDs)
      * @param node the entity caps node#ver
      * @param online indicates if the given user is online
-     * @see UserCapsNodeListener#userCapsNodeRemoved(String, String, boolean)
      */
-    public void userCapsNodeRemoved(String user, String node, boolean online)
+    public void userCapsNodeChanged(String user, ArrayList<String> fullJids,
+        String node, boolean online)
     {
         OperationSetPresence opsetPresence
-            = parentProvider.getOperationSet(OperationSetPresence.class);
-
-        if (opsetPresence != null)
-        {
-            String jid = StringUtils.parseBareAddress(user);
-            Contact contact = opsetPresence.findContactByID(jid);
-
-            // If the contact isn't null and is online we try to discover the
-            // new set of operation sets and to notify interested parties.
-            // Otherwise we ignore the event.
-            if (contact != null)
+        = parentProvider.getOperationSet(OperationSetPresence.class);
+        if (opsetPresence != null) {
+            if(LibJitsi.getConfigurationService()
+                .getBoolean(
+                    XMPPConfigForm.PROP_XMPP_USE_ALL_RESOURCES_FOR_CAPABILITIES,
+                    XMPPConfigForm.USE_ALL_RESOURCES_FOR_CAPABILITIES_DEFAULT))
```

Comment:
I read up on the bundle convention in the Jitsi projects. So if I understand correctly, references like this need to be isolated to the respective bundle the class itself resides in. In this particular case I'd simply duplicate the constant to *OperationSetContactCapabilitiesJabberImpl*, since there isn't a more appropriate place for this.